index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+require('dotenv').config();
+const server = require('./src/server');
+const PORT = process.env.PORT || 3000;
+
+
+// mongo set up 
+const MONGO_URI = process.env.MONGO_URI;
+const mongoose = require('mongoose');
+const options = {useNewUrlParser: true, userUnifiedTopology: true};
```

Comment:
This is one line that is causing issues you have: 
`userUnifiedTopology: true`
and it should be:
`useUnifiedTopology: true`